src/isonline.js

```diff
@@ -31,12 +35,16 @@ function isOnlineCli (...urls) {
           }
 
           console.log(entry, msg);
-          resolve(results);
+          resolve(msg);
```

Comment:
not sure if you want to change the programmatic api in this place. the intention here is to return json and not a human readable textstring, as it is harder to process from consumers. teh return value of the promise is the programmatic api for http://robertkowalski.github.io/nmo/api/nmo-isonline.html